feeds/wifi-trunk/hostapd/patches/901-hapd-ubus-channel-switch.patch

```diff
@@ -140,8 +140,8 @@ Index: hostapd-2020-06-08-5a8b3662/src/ap/ubus.h
  };
  
 +enum hostapd_ubus_chan_event_reason {
-+	HOSTAPD_UBUS_HIGH_INTERFERENCE,
-+	HOSTAPD_UBUS_DFS_SWITCH
++	HOSTAPD_UBUS_DFS_SWITCH,
```

Comment:
What does this change do?

Reply:
this is to align with how the enum is defined in the protobufs